src/kirby/components/list/helpers/list-swipe-action.ts

```diff
@@ -0,0 +1,12 @@
+import { ThemeColor } from '../../../helpers/theme-color.type';
+
+export interface ListSwipeAction {
```

Comment:
Remember to export this in the file `../src/kirby/list.ts`, so users of kirby can import it.